libs/backend/feature/role/src/lib/roles.controller.ts

```diff
@@ -37,7 +36,7 @@ export class RolesController {
   }
 
   @Get('/asso/:id')
-  findAll(@Param('id') id: string): Promise<RoleDto[]> {
+  public findAll(@Param('id') id: string): Promise<RoleDto[]> {
     return this.rolesService.findAll(+id);
   }
```

Comment:
Pourquoi avoir rajouter le "public" devant mais pas partout ?

Reply:
Is ok, on va faire une convention après ça pour public partout et réordonner proprement les fonctions ud back pour que ce soit ordonné :)